### ISSUE TITLE
Windows release contains a linux binary

### ISSUE THREAD

**Issue by datamuc** [QubitProducts/exporter_exporter#57]
The exe contained in https://github.com/QubitProducts/exporter_exporter/releases/download/v0.4.0/exporter_exporter-0.4.0.windows-amd64.zip is not a windows executable:

```
$ unzip ../exporter_exporter-0.4.0.windows-amd64.zip
Archive:  ../exporter_exporter-0.4.0.windows-amd64.zip
  inflating: build/exporter_exporter-0.4.0.windows-amd64/exporter_exporter.exe
$ file build/exporter_exporter-0.4.0.windows-amd64/exporter_exporter.exe
build/exporter_exporter-0.4.0.windows-amd64/exporter_exporter.exe: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, Go BuildID=-Q9NPMunilMXaOGIuByt/_eiyrk3CQNfYU62rUW7n/VKYarLwOWrQMst-s20fd/BrW2dD2LRqFZcSBwgZze, not stripped
```

**Comment by datamuc** [comment]
https://github.com/QubitProducts/exporter_exporter/blob/54fcd9b6c9556c40919839755264920670961a0a/Makefile#L107

hardcoding GOOS=windows there seems to help.

**Comment by tcolgate** [comment]
Sorry for the long delay on this, I'll look into it ASAP.

**Comment by tcolgate** [comment]
Sorry for the even longer delay, this should be properly fixed and 0.4.2 features an actual windows binary.
